acquia_lift.module

```diff
@@ -565,7 +533,7 @@ function acquia_lift_target_access($agent) {
  * Implements hook_personalize_agent_presave().
  */
 function acquia_lift_personalize_agent_presave($agent) {
-  if (!in_array($agent->plugin, array('acquia_lift', 'acquia_lift_simple_ab'))) {
```

Comment:
Let's go ahead and convert this to use acquia_lift_is_testing_agent().  Will be clearer for the next round of changes.